.github/workflows/release.yml

```
@@ -0,0 +1,74 @@
+          -DreleaseVersion=${{ github.event.inputs.releaseVersion }} \
+          -DdevelopmentVersion=${{ github.event.inputs.developmentVersion }} \
+          -DtagNameFormat=v@{project.version}
+      - name: Perform Release
+        run: mvn release:perform
+      - name: Push changes
+        run: |
+          git push origin main
+          git push origin --tags
+      - name: Get changelog entry
+        id: changelog
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2129:style:2:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)